group-coordinator/src/test/java/org/apache/kafka/coordinator/group/assignor/OptimizedUniformAssignmentBuilderTest.java

```diff
@@ -253,13 +252,15 @@ public void testValidityAndBalanceForLargeSampleSet() {
             ));
         }
 
-        Map<String, AssignmentMemberSpec> members = new TreeMap<>();
+        List<Uuid> subscribedTopics = new ArrayList<>(topicMetadata.keySet());
```

Comment:
nit: This change does not seem necessary. We start from a Set, convert it to a List, and finally convert it back to a Set.